18F/identity-idp
# Update cssbundling-rails to resolve redundant Yarn install

## 🛠 Summary of changes

Updates `cssbundling-rails` gem dependency to the latest version, from 1.0.0 to 1.4.0.

This resolves an issue where `yarn install` would be called redundantly during deployments and in continuous integration during the `assets:precompile` task. We intend to skip Yarn install during this task in both continuous integration ([source](https://github.com/18F/identity-idp/blob/ea72427c62eca25e641d0501304dc5505257ae27/.gitlab-ci.yml#L81)) and deploys ([source](https://github.com/18F/identity-idp/blob/ea72427c62eca25e641d0501304dc5505257ae27/deploy/build-post-config#L21)) since it is done as part of a previous step ([source](https://github.com/18F/identity-idp/blob/ea72427c62eca25e641d0501304dc5505257ae27/.gitlab-ci.yml#L92), [source](https://github.com/18F/identity-idp/blob/ea72427c62eca25e641d0501304dc5505257ae27/deploy/build#L27-L34)). However, both `jsbundling-rails` and `cssbundling-rails` both include a built-in `yarn install`, and while the environment variable was enough to prevent the install in `jsbundling-rails`, it was not supported as of the version of `cssbundling-rails` we had installed previously. This was identified via troubleshooting at https://github.com/rails/jsbundling-rails/pull/150#issuecomment-2050168880, and is supported as of [`cssbundling-rails@1.3.0`](https://github.com/rails/cssbundling-rails/releases/tag/v1.3.0).

## 📜 Testing Plan

1. Run `SKIP_YARN_INSTALL=true rails assets:precompile`
2. Verify that the first command you see is `yarn run` and not `yarn install`
